tests/unit/src/test/scala/tests/TreeViewSlowSuite.scala

```diff
@@ -0,0 +1,136 @@
+package tests
+
+object TreeViewSlowSuite extends BaseSlowSuite("tree-view") {
```

Comment:
This test suite doesn't stress yet the "did change" notifications. I'm not sure yet if it's worth the effort, we similarly don't have many automated tests for features like the doctor. I feel like it might be good to test a few workflows however:

- importing a build triggers a reload on project and library explorers
- compiling a source triggers a reload on the project explorer for that given build target
- dynamic compile progress updates should not be sent when the compile view is not visible